tests/integration/Web3Names.spec.ts

```diff
@@ -172,6 +172,23 @@ describe('When there is an Web3NameCreator and a payer', () => {
     )
     await submitTx(authorizedTx, paymentAccount)
   }, 40_000)
+
+  it('should be possible to reclaim a name after a w3n has been reclaimed by a payment account', async () => {
```

Comment:
looks good, I'm wondering why we don't actually check if the name has been removed on the original test (line 152)